pegasus/sites.v3/code.org/public/cookies/index.haml

```diff
@@ -0,0 +1,8 @@
+---
+title: Code.org Cookie Notice
+theme: responsive
+---
+
+%link{ :href => "/css/cookies.css", :rel => "stylesheet" }
+
+= render inline: File.read(deploy_dir('shared/partials/cookie_notice.partial')), type: :partial
```

Comment:
I'm curious why you use `deploy_dir` here but `Rails.root.join` in other places?

Reply:
`deploy_dir` finds the root path in pegasus, `Rails.root.join` finds the root path in studio. This was a weird gotcha I ran into, `Rails.root.join` works fine locally for both pegasus and dashboard because of how we run the dashboard-server locally, but in all other environments pegasus does not have access to the `Rails.root`.

Reply:
Does `deploy_dir` work for both Pegasus and Dashboard? IMO, `deploy_dir('shared/haml/onetrust_cookie_scripts.haml')` is easier to read than 
`Rails.root.join('..', 'shared', 'haml', 'onetrust_cookie_scripts.haml')`.  
This is just an observation, not a suggestion to change.